lib/jormungandr/test/unit/Cardano/Wallet/Jormungandr/TransactionSpec.hs

```diff
@@ -529,23 +531,27 @@ goldenTestDelegationCertTx tl keystore pool (accountXPrv, pass) inps outs bytes'
 xprvSeqFromSeed
     :: ByteString
     -> (ShelleyKey depth XPrv, Passphrase "encryption")
-xprvSeqFromSeed seed =
-    ( Seq.unsafeGenerateKeyFromSeed (Passphrase (BA.convert seed), mempty) pwd
+xprvSeqFromSeed bytes =
+    ( Seq.unsafeGenerateKeyFromSeed (seed, Nothing) pwd
     , pwd
     )
   where
     pwd = mempty
+    seed = SomeMnemonic $ entropyToMnemonic @12 ent
+    ent = either (error . show) id $ mkEntropy @(EntropySize 12) bytes
 
 xprvRndFromSeed
     :: ByteString
     -> (ByronKey 'AddressK XPrv, Passphrase "encryption")
-xprvRndFromSeed seed =
-    ( Rnd.unsafeGenerateKeyFromSeed derPath (Passphrase $ BA.convert seed) pwd
+xprvRndFromSeed bytes =
+    ( Rnd.unsafeGenerateKeyFromSeed derPath seed pwd
     , pwd
     )
   where
     pwd = mempty
     derPath = (minBound, minBound)
+    seed = SomeMnemonic $ entropyToMnemonic ent
+    ent = either (error . show) id $ mkEntropy @(EntropySize 12) bytes
```

Comment:
I've seen this probably 10 times in this PR. I think that's a good case for a `unsafeMkEntropy` with a proper call stack to be defined in our unsafe module.